### PR TITLE
:bug: revert placement decision start index from 1 to keep backward compatibility

### PR DESCRIPTION
**Pull request by haoqing0110**
## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/placement/controllers/scheduling/scheduling_controller.go [hunk 1]
```diff
@@ -465,7 +465,8 @@ func (c *schedulingController) generatePlacementDecisionsAndStatus(
 
 	// generate placement decision for each decision group
 	for decisionGroupIndex, decisionGroup := range decisionGroups {
-		// generate placement decisions and status, group and placement name index starts from 0.
+		// generate placement decisions and status, group name index starts from 0, placement name index
+		// starts from 1 to keep backward compatibility.
```
Comment (qiujian16) [inline pkg/placement/controllers/scheduling/scheduling_controller.go:469]: add a todo here, we should be consistent with index or using a random generate name when version bumps

Comment (haoqing0110) [reply pkg/placement/controllers/scheduling/scheduling_controller.go]: added

**Pull request merged**